mindjunkies/forums/migrations/0001_initial.py

```
@@ -249,4 +249,4 @@ class Migration(migrations.Migration):
                 on_delete=django.db.models.deletion.CASCADE, to="forums.reply"
             ),
         ),
-    ]
+    ]
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)